src/main/resources/application.yml

```diff
@@ -74,6 +72,16 @@ workspace:
     username: ${env.db.ws.user}
 
 terra.common:
+  kubernetes:
+    in-kubernetes: true
+  stairway:
+    cluster-name-suffix: workspace-stairway
+    force-clean-start: false # ${env.db.init}
+    max-parallel-flights: 50
```

Comment:
is 50 enough? It a bigger number in `buffer`. @ddietterich

Reply:
I have no reason to think we need more than 50. We run with 3 pods, so that is a net of 150 flights running in parallel. We are likely to hit the connection limit on the underlying database before we run out of threads.

I have raised concerns with you before @yonghaoy about the very high number of threads you specify and the memory consequences. But so long as you have nodes able to support your memory use, it should work.